### DISCUSSION TITLE
What will happen to Info if i drop the Torrent in another goroutine?

### DISCUSSION THREAD

**Discussion by ghost** [anacrolix/torrent#502] [Q&A]
So, in one goroutine, i listen for GotInfo() because i want to start the torrent(setting priority of files to normal) after Info is recieved.

So,during listening of GotInfo() of torrent, if i Drop() the torrent in another goroutine, what will happen to Info() ? will GotInfo() ? should i listen for torrent.Closed() too?

**Comment by anacrolix** [comment]
You should listen for torrent.Closed too.
